utils/send-mail.js

```diff
@@ -0,0 +1,12 @@
+
+const sgMail = require('@sendgrid/mail');
+
+sgMail.setApiKey(process.env.SEND_GRID_API_KEY);
+const msg = {
+  to: 'acmucsdmailing@gmail.com',
+  from: 'acmucsdmailing@gmail.com',
+  subject: 'Hello ACM User! Please confirm you email here',
+  text: 'Click here to verify your email',
+  html: '<a href="#">Click here to verify your email</a>',
+};
+sgMail.send(msg);
```

Comment:
looks like this returns a promise ([source](https://github.com/sendgrid/sendgrid-nodejs/blob/master/packages/mail/src/classes/mail-service.js#L144)) so maybe `return sgMail.send(msg)` in the function?